src/saltext/vmware/states/storage_policy.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+import saltext.vmware.utils.connect as connect
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "vsphere_storage_policy"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def storage_policy(name, storagePolicies):
+    """
+    Checks state of storage policy of each host
+    policies
+        Policy list to set state to
+    """
+
+    res = connect.request(
+        "/api/vcenter/storage/policies", "GET", opts=__opts__, pillar=__pillar__
+    )
+    response = res["response"].json()
+    current_policies = list(map(lambda p: p['name'], response))
```

Comment:
**suggestion** this can just be a list comprehension instead:

```suggestion
    current_policies = [p["name"] for p in response]
```

Should work exactly the same :+1:

Reply:
Done